src/dapps/Salad/containers/DepositForm/DepositForm.scss

```diff
@@ -1,93 +1,96 @@
 @import '~@/scss/GlobalVariables';
-
-.salad-deposit-content {
-  padding: 40px;
-
-  // Tablet
-  @media all and (min-width: calc(#{$mobile-width} + 1px)) and (max-width: $tablet-width) {
-    padding: 20px;
-  }
-
-  // Mobile
-  @media all and (max-width: $mobile-width) {
-    padding: 10px;
-  }
-}
-
-.salad-deposit-form {
-  padding: 20px;
-
-  // Tablet
-  @media all and (min-width: calc(#{$mobile-width} + 1px)) and (max-width: $tablet-width) {
-    padding: 20px;
-  }
-
-  // Mobile
-  @media all and (max-width: $mobile-width) {
-    padding: 10px;
-  }
-
-  .deliveryAddress {
-    @media all and (max-width: 767px) {
-      margin-top: 10px !important;
+#salad-deposit-container {
+    padding: 0px;
+    font-family: 'Avenir Next';
+    font-size: 18px;
+    line-height: 25px;
+    font-weight: normal;
+    // Tablet
+    @media all and (min-width: calc(#{$mobile-width} + 1px)) and (max-width: $tablet-width) {
+        padding: 20px;
     }
-  }
-  .deliveryAddress-label {
-    font-weight: 400;
-    margin-top: 10px !important;
-    margin-bottom: 2px !important;
-  }
-  .deposit-header {
-    font-weight: 400;
-    margin-top: 1px !important;
-    margin-bottom: 20px !important;
-  }
-
-  .mixAmount-label {
-    font-weight: bold;
-  }
-
-  .err-msg {
-    margin-bottom: 2px;
-    color: $red-1;
-    font-weight: 600;
-  }
-
-  .deposit-container {
-    
-
-    .submit-btn {
-      width: 100%;
-      height: 52px;
-      border-radius: 4px;
-      background-color: #05C0A5;
+    // Mobile
+    @media all and (max-width: $mobile-width) {
+        padding: 10px;
     }
-
-    .deliveryAddress-input {
-      display: flex;
-      width: 100%;	
-      @media all and (max-width: $tablet-width) {
-        display: block;
-        text-align: center;
-      }
-
-      input {
-        background-color: $light-grey-1;
-        border: 2px solid #05C0A5;
-        height: 52px;
-        padding-left: 20px;
-        padding-right: 20px;
-        width: 100%;
-
-        @media all and (max-width: $tablet-width) {
-          border-radius: 4px;
+    #deposit-info {
+        position: absolute;
```

Comment:
Any reason it's absolute? This combined with the left/top puts it out of screen and unresponsive to screen changes.  Will propose a fix in my branch.

Reply:
Ah okay. No, I was going by the Figma specs and a newbie at this styling ;-).